### PR TITLE
Py3 prints

### PR DESCRIPTION
**Pull request by zuphilip**
The `tsht`-tests works for me locally with Python 2 as well as Python 3 and also Travis agreed. However, let us check this carefully and test some more examples.

### CODE REVIEW

**Comment by stweil** [comment]
`2to3` does what the name says, namely convert code which was written for Python 2 to code which works with Python 3. It does not promise that it will still work with Python 2 after the conversion. Even if it seems to work at a first glance, here is an example of the problems with Python 2:

    $ ./hocr-check -h
    ('usage:', './hocr-check', '[-o] file.html')

In Python 2 `print` is not a function, so the output includes everything which follows the `print` keyword which is not what we expect here.

I only see two possible solutions to support both versions of Python:

* Conditional code, that means one print statement for each version.
* Additional wrapper library. This is the commonly used solution.

**Comment by zuphilip** [comment]
Ah, okay, thank you for the explanation. [One can include in every file with a print statement at the very beginning the statement](http://python-future.org/compatible_idioms.html#print):
```py
from __future__ import print_function
```
I guess this is the wrapper library you mean?

**Comment by stweil** [comment]
Yes.

File: hocr-combine [hunk 1]
```diff
@@ -35,4 +36,4 @@ for fname in sys.argv[2:]:
         page = doc.importNode(page,1)
         container.append(page)
 
-print(etree.tostring(doc, pretty_print=True))
+print((etree.tostring(doc, pretty_print=True)))
```
Comment (zuphilip) [inline hocr-combine]: I guess this is not needed.

**Comment by zuphilip** [comment]
Okay, I added the wrapper library to the files. Thus, we can test and discuss this further.

File: hocr-combine (hunk 1 above)
Comment (stweil) [inline hocr-combine]: Yes, it should be removed.

File: hocr-merge-dc [hunk 2]
```diff
@@ -61,4 +62,4 @@ for node in dc_nodes:
         hnode.attrib['content'] = value
         hocr_meta.append(hnode)
 
-print(etree.tostring(hocr_doc, pretty_print=True))
+print((etree.tostring(hocr_doc, pretty_print=True)))
```
Comment (stweil) [inline hocr-merge-dc]: This one is also not needed.

File: hocr-pdf [hunk 3]
```diff
@@ -136,6 +137,6 @@ CMGjwvxTsr74/f/F95m3TH9x8o0/TU//N+7/D/ScVcA=
 
 if __name__ == "__main__":
   if len(sys.argv) == 1:
-    print("Usage: %s <imgdir>\n" % os.path.basename(sys.argv[0]))
+    print(("Usage: %s <imgdir>\n" % os.path.basename(sys.argv[0])))
```
Comment (stweil) [inline hocr-pdf]: Not needed, too.

**Comment by zuphilip** [comment]
I deleted the three unnecessary brackets. Please have a look at the new version.

**Comment by stweil** [comment]
@zuphilip, I changed the order of the commits and squashed two of them. This avoids unnecessary commits and asserts that the single commits don't break anything. I think that the PR is now ready to get merged.

**Comment by zuphilip** [comment]
Fine for me! Thanks for squashing the commits and reviewing.

**Comment by stweil** [comment]
Thank you for this PR. Merged now.

**Pull request merged**